### PR TITLE
ISSUE-63: Upgrade from RC3 to 1.0.0 doc needs edits

### PR DESCRIPTION
**Pull request by aksm**
Resolves part of #63

### CODE REVIEW

File: docs/upgradeFromRC3.md [hunk 1]
```diff
@@ -115,43 +115,36 @@ Time to fetch the `1.0.0` branch and update our `docker-compose` and `composer`
...
-<!--switch_below
+`IMPORTANT NOTE`: For AWS EC2. If you selected an `IAM role` for your server when setting it up/deploying it, `min.io` will use the AWS EC2-backed internal API to request access to your S3. This means the ROLE itself needs to have read/write access (ACL) to the given Bucket(s) and your key/secrets won't be able to override that. Please do not ignore this note. It will save you a LOT of frustration and coffee. You can also run an EC2 instance without a given IAM and in that case just the ACCESS_KEY/SECRET will matter.
 
-??? info "OSX (macOS)/x86-64"
+Now that you know, you also know that these values **should not be shared** and this `.env` file **should not be committed/kept in version control**. Please be careful.
 
-    ```shell
-    cp docker-compose-osx.yml docker-compose.yml
-    ```
+Then copy the appropriate `docker-compose` file for your architecture:
```
Comment (DiegoPino) [inline docs/upgradeFromRC3.md:134]: @aksm a step is missing here. The user should always backup the original docker-compose.yml. Most of our users end customizing/adding/removing configs and when doing the copy they overwrite all their particularities. Also a git diff example would be great

File: docs/upgradeFromRC3.md [hunk 2]
```diff
@@ -217,6 +210,7 @@ Important here is the `STATUS` column. It **needs** to be a number that goes up
 ### Step 3:
 
 Instead of using the provided `composer.lock` out of the box we are going to loosen certain dependencies and bring manually Archipelago modules, all this to make update easier and future upgrades less of a pain.
```
Comment (DiegoPino) [inline docs/upgradeFromRC3.md]: We provide now a composer.lock.default. We also need to be sure that the user doing this runs all composer require against the original *(their)* composer.lock and not against the new default one

**Review by DiegoPino — CHANGES_REQUESTED** [review]
Some changes are needed. Check slack for what happened to our friends at WWU last week

File: docs/upgradeFromRC3.md (hunk 1 above)
Comment (aksm) [reply docs/upgradeFromRC3.md:134]: Ok, does this look ok? 247c2be75e5d35e1984948e3cdef4008a1eaeec6

File: docs/upgradeFromRC3.md (hunk 2 above)
Comment (aksm) [reply docs/upgradeFromRC3.md]: And does this make sense? 8a7361c1df8a1b7098b0ba2eb1d063b5d429ff04

**Review by DiegoPino — APPROVED** [review]

**Pull request merged**